package.json

```diff
@@ -17,7 +17,7 @@
     "fetch-examples-gg": "cross-env NODE_ENV=development node ./server/scripts/fetch-examples-gg.js",
     "fetch-examples:prod": "cross-env NODE_ENV=production node ./dist/fetch-examples.bundle.js",
     "fetch-examples-gg:prod": "cross-env NODE_ENV=production node ./dist/fetch-examples-gg.bundle.js",
-    "heroku-postbuild": "touch .env; npm run build"
+    "heroku-postbuild": "touch .env; echo \"MONGO_URL=$MONGO_URI\nMAILGUN_KEY=$MAILGUN_API_KEY\n\" >> .env; npm run build"
```

Comment:
i like this solution, and like that it's clear that it's only for heroku deployment. just want to confirm that is works—`npm run start:prod` assumes that the environment variables are already loaded into the shell, and not in a `.env` file.

Reply:
Yes, Heroku will provision the add-ons first (Mailgun, Mongo), which will set the environment variables before running any commands.